apps/src/templates/projects/ProjectCard.jsx

```diff
@@ -105,6 +108,11 @@ export default class ProjectCard extends React.Component {
       Object.assign(thumbnailStyle, styles.fullThumbnail);
     }
 
+    const noTimeOnCardStyle = {};
```

Comment:
lines 111-113 could simplify to:

```
const cardStyle = this.props.isDetailView ? {} : styles.noTime;
```

which makes the `<div style={cardStyle}>` more explicit without needing extra context